src/status_im/common/lightbox/top_view.cljs

```diff
@@ -114,6 +94,6 @@
        {:active-opacity      1
         :accessibility-label :image-options
         :on-press            #(rf/dispatch [:show-bottom-sheet
```

Comment:
perhaps it's cleaner that we just pass an on-press event?
and then leave this behaviour up to the page to handle?